test/render-ctmpls.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+env=$1
+vault_token=$2
+
+printf "Creating Lira config\n"
+docker run -it --rm -v $PWD:/working -e VAULT_TOKEN=$vault_token \
```

Comment:
Could we pass in a parameter for the working directory, instead of hard coding `$PWD`? I think that flexibility could be useful.

Reply:
sure thing